applications/crossbar/src/modules/cb_cdrs.erl

```diff
@@ -24,6 +24,10 @@
          ,to_csv/1
         ]).
 
+-ifdef(TEST).
+-export([group_cdrs/1]).
```

Comment:
You know what we could do? Add `+export_all` to the Makefile target that builds tests. This way it would be less painful to write tests for any function. It would also make it kind of less clear to know which functions have tests though…